components/feature/search/src/main/assets/search/list.json

```diff
@@ -593,10 +614,17 @@
         ]
       }
     },
+    "nl-NL": {
+      "default": {
+        "visibleDefaultEngines": [
+          "google-b-m", "bing", "bolcom-nl", "ddg", "wikipedia-nl", "amazon-nl", "ebay-nl"
+        ]
+      }
+    },
```

Comment:
I wonder why we don't make this change in the `nl` block?

Reply:
I learned that lots of countries speaks Dutch (`nl`) but not in Netherlands.  Since amazon.nl and ebay.nl is country based and not language based, we need to make sure that it only apply to people who speaks Dutch in Netherlands.

Reply:
Is `nl-NL` a regional dialect, or locale and region combined? Desktop used to use a slightly different format for handling regions: https://searchfox.org/mozilla-esr78/rev/2dc99291ebe1d73e98ae7fd73cd971b1c3e97bb1/browser/components/search/extensions/list.json#54-88

Reply:
Yes, the `nl-` is for Dutch speakers.  The `-NL` is for Netherlands.  Adding to `nl-NL` means only people who set their device Dutch in Netherlands will have `eBay.nl` as a search engine option.